crates/editor/src/editor_tests.rs

```diff
@@ -3117,7 +3117,7 @@ fn test_duplicate_line(cx: &mut TestAppContext) {
                 DisplayPoint::new(3, 0)..DisplayPoint::new(3, 0),
             ])
         });
-        view.duplicate_line(&DuplicateLine::default(), cx);
+        view.duplicate_line_down(&DuplicateLineDown::default(), cx);
```

Comment:
NIT: do you really need those `::default()` now given that both `Duplicate*` are field-less structs now?